public/style.css

```diff
@@ -19,28 +19,35 @@ div {
 	padding: 35px;
 }
 
-.review-interactor {
-	width: 730px;
-  height: auto;
-  padding-top: 25px;
-  background-image: url(https://lululemon.ugc.bazaarvoice.com/static/7834redes/dottedBorder.gif);
-  background-position-x: left;
-  background-position-y: top;
```

Comment:
The vertical background position here (background-position-y) is set to "top" which places the background at the "top" of the element. Is this the behavior you want for this element? Could the parent element of your review text-box have this background and could you set the background position y to "bottom" instead?